content/en/storage-providers/seal-workers/post-workers.md

```diff
@@ -0,0 +1,145 @@
+export FIL_PROOFS_MAXIMIZE_CACHING=1
+export FIL_PROOFS_USE_GPU_COLUMN_BUILDER=1 # when GPU is available
+export FIL_PROOFS_USE_GPU_TREE_BUILDER=1   # when GPU is available
+export FIL_PROOFS_PARAMETER_CACHE=/fast/disk/folder # > 100GiB!
+export FIL_PROOFS_PARENT_CACHE=/fast/disk/folder2   # > 50GiB!
+```
+
+The PoSt workers will fail to start if the file descriptor limit is not set high enough. Raise the the file descriptor limit temporarily before running with `ulimit -n 1048576` or permanently by following the [Permanently Setting Your ULIMIT System Value](https://github.com/filecoin-project/lotus/discussions/6198) guide.
+
+{{< alert icon="tip" >}}
+When initially fetching parameter files, remember to set the [`IPFS_GATEWAY` variable when running from China]({{< relref "nodes-in-china" >}})
+{{< /alert >}}
+
+### Run the PoSt worker
+
+```shell
+lotus-worker run <flags>
+```
+
+The above command will start the worker. You´ll need to specify which PoSt operation you want the worker to perform with one of the following flags set to true.
+
+```
+   --winningpost                 enable winning post (default: false)
+   --windowpost                  enable window post (default: false)
+```
+
+A PoSt worker instance can only be either a winningPoSt worker, or a windowPoSt worker.
+
+When a winningPoSt or windowPoSt worker connects to the _lotus-miner_, the lotus miner will delegate all winningPoSt or windowPoSt tasks to that worker. If both tasks are delegated to seperate PoSt workers, no PoSt tasks will be executed locally on the miner instance. If a worker is stopped, the lotus-miner instance switches back to local PoSt automatically.
+
+You can verify that PoSt workers are connected to the lotus-miner with `lotus-miner proving workers`:
+
+```shell
+$ lotus-miner proving workers
+Worker 0, host computer
+        CPU:  [                                                                ] 0 core(s) in use
+        RAM:  [||||||||||||||||||                                              ] 28% 18.1 GiB/62.7 GiB
+        VMEM: [||||||||||||||||||                                              ] 28% 18.1 GiB/62.7 GiB
+
+Worker 1, host othercomputer
+        CPU:  [                                                                ] 0 core(s) in use
+        RAM:  [||||||||||||||                                                  ] 23% 14 GiB/119.2 GiB
+        VMEM: [||||||||||||||                                                  ] 23% 14 GiB/119.2 GiB
+        GPU: GeForce RTX 2080, not used
+```
+
+You can also verify that PoSt workers are connected in the workers-section of `lotus-miner info`.
+
+```shell
+$ lotus-miner info
+Workers:
+	Seal: 15
+	WdPoSt: 4
+	WinPoSt: 3
+```
+
+### Optional settings
+
+Although the default settings are reasonable you can configure some advanced settings when running the PoSt workers, that can be tested for local optimizations of your hardware.
+
```

Comment:
Could this result in delayed windowPost? If yes, can we add the warning for the same? Since it could lead to slashing.